connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConfigTransformerTest.java

```diff
@@ -53,64 +50,72 @@ public class WorkerConfigTransformerTest {
     public static final String TEST_RESULT_WITH_TTL = "testResultWithTTL";
     public static final String TEST_RESULT_WITH_LONGER_TTL = "testResultWithLongerTTL";
 
-    @Mock private Herder herder;
-    @Mock private Worker worker;
-    @Mock private HerderRequest requestId;
+    private final Herder herder = Mockito.mock(Herder.class);
+    private final Worker worker = Mockito.mock(Worker.class);
+    private final HerderRequest requestId = Mockito.mock(HerderRequest.class);
```

Comment:
I decided to go for `@RunWith(MockitoJUnitRunner.class)` in the newest commit, but I am happy to change it.

Reply:
Since we are in the process of migrating to JUnit 5, it's simpler if we don't rely on JUnit 4 specific features. It's fine to use the `mock` method, but please use static imports to make it more concise.

Reply:
It's pretty straightforward to replace `@RunWith(MockitoJUnitRunner.class)` with `@ExtendWith(MockitoExtension.class)`, and we're already using that approach in other modules (such as [here](https://github.com/apache/kafka/blob/badfbacdd09a9ee8821847f4b28d98625f354ed7/core/src/test/java/kafka/test/ClusterTestExtensionsTest.java#L36)). We also get to leverage things like the [`@MockitoSettings` annotation](https://www.javadoc.io/doc/org.mockito/mockito-junit-jupiter/2.19.1/org/mockito/junit/jupiter/MockitoSettings.html) if we stick to this style.

@ijuma since it's only going to be a one-line (well, excluding imports) change per test, do you think we could stick with `@RunWith` for now in our Mockito tests?

Reply:
Yeah, that's fine.